docker/hadoop/common/startup/51_postgres_init

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -ex
+
+# init database for metastore
+
+#rm -rf /var/lib/mysql
```

Comment:
Can we remove the commented lines please?

Reply:
good catch, forgot about those